plugin/src/test/resources/ide/language/java/Docs.java

```diff
@@ -59,3 +59,15 @@ Integer badFunction(T member) {
         return 1; // It <warning descr="IT_VBZ">are</warning> <warning descr="MORFOLOGIK_RULE_EN_US">eror</warning> comment
     }
 }
+
+/**
```

Comment:
We've decided to add Russian only for docs? I would add it also to string literals

Reply:
Yeah, we only need to check that russian rules works